packages/mui-material/src/Autocomplete/Autocomplete.js

```diff
@@ -582,6 +582,106 @@ const Autocomplete = React.forwardRef(function Autocomplete(inProps, ref) {
   const popperSlotProps = slotProps.popper ?? componentsProps.popper;
   const popupIndicatorSlotProps = slotProps.popupIndicator ?? componentsProps.popupIndicator;
 
+  const renderPopper = () => {
```

Comment:
```suggestion
```
Instead of using a function, just directly use the `container` variable and use it like this:
```jsx
{anchorEl ? container : null}
```